src/node.cc

```diff
@@ -4929,28 +4936,29 @@ int Start(int argc, char** argv) {
   v8_initialized = true;
 
 #if ENABLE_TTD_NODE
-  bool chk_debug_enabled = debug_options.debugger_enabled() || debug_options.inspector_enabled();
+  bool chk_debug_enabled = debug_options.debugger_enabled()
+                           || debug_options.inspector_enabled();
```

Comment:
I don't remember why I think this, but I thought there was a standard somewhere to place the operator at the end of the last line.  If that isn't the case, just ignore me.